### DISCUSSION TITLE
Panel Persistent Groups

### DISCUSSION THREAD

**Discussion by n1x9** [mbnuqw/sidebery#1088] [Ideas]
Good day,

Is it possible to have groups persist within panels any time Firefox is opened with Sidebery installed? Currently if I close the browser, the next time I start it any Sidebery panel groups I created are gone.

**Comment by albino1** [comment]
Are you still on Sidebery v4 from the Mozilla addon store?  If so, there's a bug in there that can cause that.  Most people have migrated to v5 which is still technically in beta, but very stable.  You can find the latest version here:

https://github.com/mbnuqw/sidebery/releases/tag/v5.0.0b33

**Reply by n1x9** [reply]
Thanks for the feedback! I have upgraded to the Beta version. However my panel groups are not persisting between Firefox restarts. Is there some configuration setting I need to change for the panel groups to be saved?

**Reply by albino1** [reply]
Hmm, I'm not sure.  Somebody more qualified will have to come along and try and help :)

**Reply by Hikandy** [reply]
> Thanks for the feedback! I have upgraded to the Beta version. However my panel groups are not persisting between Firefox restarts. Is there some configuration setting I need to change for the panel groups to be saved?

were you able to figure this out? I'm in the same boat
